Dockerfile.konflux

```diff
@@ -0,0 +1,57 @@
+
+#ARG BUILD_BASE=develop
+
+#go-1.21
+FROM registry.redhat.io/ubi8/go-toolset:1.21@sha256:742ae6ec1aef3e7faae488c47695fb64964d342aefecf52d23bd9d5e6731d0b6 AS build
+
+USER root
+
+# needed for konflux as the previous stage is not used
+WORKDIR /opt/app
+COPY go.mod go.sum ./
+# Download dependencies before copying the source so they will be cached
+RUN go mod download
+
+# Copy the source
+COPY . ./
+
+USER root
+
+# Build the binaries using native go compiler from BUILDPLATFORM but compiled output for TARGETPLATFORM
+# https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/
+RUN go build -o puller model-serving-puller/main.go && \
+    go build -o triton-adapter model-mesh-triton-adapter/main.go && \
+    go build -o mlserver-adapter model-mesh-mlserver-adapter/main.go && \
+    go build -o ovms-adapter model-mesh-ovms-adapter/main.go && \
+    go build -o torchserve-adapter model-mesh-torchserve-adapter/main.go
+
+
+#Copy build assets to create the smallest final runtime image
+###############################################################################
+
+#ubi-minimal:latest
+FROM registry.redhat.io/ubi8/ubi-minimal@sha256:cf095e5668919ba1b4ace3888107684ad9d587b1830d3eb56973e6a54f456e67 as runtime
+
+ARG USER=2000
+
+USER root
+
+
+# Copy over the binary and use it as the entrypoint
+COPY --from=build /opt/app/puller /opt/app/
+COPY --from=build /opt/app/triton-adapter /opt/app/
+COPY --from=build /opt/app/mlserver-adapter /opt/app/
```

Comment:
we are missing this https://github.com/red-hat-data-services/modelmesh-runtime-adapter/blob/rhoai-2.13/Dockerfile.konflux#L49C17-L49C84

since the python dependencies were removed from 2.13, I guess we should remove it from there as well.